projects.json

```diff
@@ -3029,15 +3029,7 @@
       {
         "action": "BuildSwiftPackage",
         "configuration": "release",
-        "tags": "sourcekit sourcekit-smoke",
-        "xfail": [
-            {
-                "issue": "Only Supports 5.9+",
-                "compatibility": "5.0",
```

Comment:
I assume this should no longer be needed now the version is correctly set to 5.9